README.md

```diff
@@ -370,15 +370,6 @@ Other Style Guides
   - [4.5](#arrays--callback-return) Use return statements in array method callbacks. It's ok to omit the return if the function body consists of a single statement following [8.2](#8.2). eslint: [`array-callback-return`](http://eslint.org/docs/rules/array-callback-return)
 
     ```javascript
-    // good
```

Comment:
@ljharb  Agreed, I guess the only other scenario would be to have `.map` perform some kind of side effects.  

I can totally go both ways on this and appreciate your feedback but would you care to have this change in the documentation where `.map` is moved to the bottom of the block?  The benefit here would be to first observe things you shouldn't do with filter/reduce followed by good practices for both, followed with straight up good practices with `.map`.

I can change the commit message/title accordingly, but if think it's unnecessary, I can close this out as well.  

Thanks for the feedback